entrypoint.sh

```diff
@@ -8,8 +8,10 @@ GITHUB_WORKFLOW_URL="https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITH
 # Create temp dir
 mkdir -p "$(dirname $LYCHEE_TMP)"
 
+ARGS="$@"
+[[ "$ARGS" =~ "--format " ]] || ARGS="$ARGS --format markdown"
```

Comment:
I believe this condition may have false-positives. Let's say 

```yaml
with:
  args: --verbose --no-progress "--format "
```

this is a valid usage right?

Reply:
That call is not accepted by lychee
```
❯ lychee --verbose --no-progress "--format "                                                                                                        
error: Found argument '--format ' which wasn't expected, or isn't valid in this context
	Did you mean --format?
```

Reply:
cool